specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,402 @@
         }
       }
     },
+    "potentialOutagesGetResult": {
+      "description": "The Get PotentialOutages operation response.",
+      "type": "object",
+      "$ref": "#/definitions/potentialoutage"
+    },
+    "potentialoutage": {
+      "type": "object",
+      "description": "Potential outages detected by Brain.",
+      "properties": {
```

Comment:
It looks like these are readyOnly fields. Consider marking them as such. Not a blocker now but might come up during the time you want to merge your swagger to master.

Reply:
thanks, I'd like to address it on this stage.